eng/pipelines/libraries/prepare-publish-for-tests.yml

```diff
@@ -0,0 +1,26 @@
+# Steps used to prepare the Artifacts Staging Directory with required files for
```

Comment:
I think it would be better to have these 2 templates as 1 template, that will make it easier to read what we're already copying and so that we don't duplicate any of these steps.